Gemfile

```diff
@@ -51,20 +51,31 @@ gem 'bugsnag'
 # markdown processing
 gem 'redcarpet'
 
+# API stackoverflow
+gem 'ruby-stackoverflow', git: 'https://github.com/PowerCodeGirls/ruby-stackoverflow.git' 
```

Comment:
It would be helpful to add a comment explaining that this is a fork that specifically sets the site to `gamedev`.